crates/meilisearch/tests/upgrade/mod.rs

```diff
@@ -64,8 +64,8 @@ async fn version_requires_downgrade() {
 #[actix_rt::test]
 async fn upgrade_to_the_current_version() {
     let temp = tempfile::tempdir().unwrap();
-    let server = Server::new_with_options(default_settings(temp.path())).await.unwrap();
-    drop(server);
+    // let server = Server::new_with_options(default_settings(temp.path())).await.unwrap();
+    // drop(server);
```

Comment:
Same here: Opening multiple Servers never worked, as the Envs were kept open anyway...
What do you think @irevoire of removing these lines?